### ISSUE TITLE
Define a clear objective for the Aragon Cooperative

### ISSUE THREAD

**Issue by yeqbfgxjiq** [aragoncoop/temp#3]
Currently, it is not clear what problem the Aragon Cooperative exists to solve. Addressing this is paramount. If you have a vision for a problem in the Aragon ecosystem that the Aragon Cooperative can solve, please create a "meta proposal" Issue describing the problem, the solution, and how the Aragon Cooperative can help. 

<br>

### For Example:

Problem:
- As a new community member there's not a lot of ways to engage beyond commenting on the forum, voting in ANVs, and starting or joining a project. 

Solution:
- A community led organization could provide support for smaller initiatives that aren't quite enough for a Nest Grant, but are cool and would contribute to the Aragon ecosystem. 

How the Aragon Cooperative can help:
- Beyond what was described in AGP-40, there's a lot more that could be done to support grass roots community initiatives and improving education/outreach in the Aragon ecosystem. The Aragon Cooperative could help build documentation, demos, and engage in workshops/meetups to help people understand what Aragon is and how they can build stuff with it. I think that this should be the mission of the Aragon Cooperative :)

Summary:
- The Aragon Cooperative functions to support grass roots community initiatives in the broader Aragon ecosystem. This will include general documentation/guides, demos, and workshops/meetups to help people understand what Aragon is and how they can build stuff with it. The main focus is engaging with people, talking to them, and distilling their questions/comments/concerns into actionable insights for the broader Aragon community.

**Comment by yeqbfgxjiq** [comment]
Now that #26 (Adopt the Temporary Aragon Cooperative Workflow as the official governance process of the Aragon Cooperative) has passed the objective is clear: deliver on AGP-40. 

Once that is accomplished, it would be good to define a more general objective like the example outlined above.

**Comment by yeqbfgxjiq** [comment]
It was recently brought to my attention by @rkzel that AGP-40 and even the example provided above addresses the "what we do?" question, but not "why we do whatever it is we are doing" question. 

With that in mind, if we approach the vision from a philosophical point of view they recommended a few resources that could help us clarify our thinking:
- https://www.eff.org/cyberspace-independence
- http://www.phrack.org/issues/7/3.html

I'd also like to add that (imho) Aragon has an awesome [manifesto](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md), and maybe it would be worth exploring something similar for the cooperative?

**Comment by sepu85** [comment]
this things has been discussed already months before your arrival to the Coop.

> I'd also like to add that (imho) Aragon has an awesome [manifesto](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md), and maybe it would be worth exploring something similar for the cooperative?

this has already being explored and partially approved by the community in [ACGP-0](https://github.com/aragoncoop/ACGP/blob/master/ACGPs/ACGP-0.md). @burrrata in case you haven't read it, please go and have a look at [this forum thread](https://forum.aragon.org/t/aragon-coop-dao-a-minimum-viable-manifesto-mvm/690).

The Manifiesto addresses the Coop value statement, we are still missing to agree on a purpose of existence (it could work to do it in terms of mission and vision).

For me it's also pretty clear that par of it's purpose (or objective as this is being called in this thread) it's about setting a proper decentralized governance framework, which I'm seeking to approach with this [excel document](https://docs.google.com/spreadsheets/d/1wUVZ6pSeaQtI2_JVn0Bsli47WA4ba939Tyl9FXXxmrE/edit?usp=sharing)

Could you have a look at it and contribute?

**Comment by yeqbfgxjiq** [comment]
@sepu85 

> this has already being explored and partially approved by the community in ACGP-0. @burrrata in case you haven't read it, please go and have a look at this forum thread.

Decisions are made via the Ethereum Mainnet DAO. There is no "partial approval." The official process for governance in the Aragon Cooperative is the process described in this repo's README.md. In case you haven't read it, please go and have a look at it.
- https://github.com/aragoncoop/temp

Opened the excel sheet. I don't see how that's a "governance framework", yet alone a "decentralized governance framework." It's an excel sheet. Also, you're comparing the Aragon Cooperative to Bitcoin. Seriously?

**Comment by yeqbfgxjiq** [comment]
Moving a potential "purpose"/objective section from [a draft of potential Aragon Cooperative bylaws](https://github.com/pythonpete32/aragoncoop_governance/blob/master/bylaws.md) to this discussion:
- "To support grassroots community initiatives in the broader Aragon ecosystem. This will include but not limited to general documentation and guides, demos, workshops, and meetups, with the purpose of helping people understand what Aragon is and how they can build with it. Its main focus is on engaging with people, talking to them, and distilling their questions and comments into actionable insights for the broader Aragon community."

**Comment by sepu85** [comment]
> Decisions are made via the Ethereum Mainnet DAO. There is no "partial approval." The official process for governance in the Aragon Cooperative is the process described in this repo's README.md.

what I intended to mean is that the community have already signaled some willings/decisions regarding this point. There's work already done. Please don't throw it away ;-)

**Comment by yeqbfgxjiq** [comment]
@sepu85 No intention to throw anything away.
- If you'd like to integrate an idea into the current governance process, the [Meta Proposal](https://github.com/aragoncoop/temp#how-do-we-modify-this-protocol) track is the way to go. 
- If you want to continue a project that works on one of the AGP-40 deliverables, a [Work Proposal](https://github.com/aragoncoop/temp#how-do-we-decide-who-does-what) is the way to go. 
- If you'd like to get paid for past work I created a Meta Proposal specifically for that [here](https://github.com/aragoncoop/temp/issues/17). 

The current model can easily satisfy integrating and supporting past and future work. That being said, the current model is based on concrete proposals and voting in the Ethereum mainnet DAO for the Aragon Cooperative. There is no "partial approval" or "rough consensus."

**Comment by calcadarafael** [comment]
From what I could gather from the beginning, the reason to create the Coop was to improve the past situation regarding voter participation and outside voices of the community, while testing 1-1 vote and trying to onboard other initiatives. The manifesto [post](https://forum.aragon.org/t/aragon-coop-dao-a-minimum-viable-manifesto-mvm/690)

I see no reason why the AGP-40 that has "deliverables" is more important than the WHY we are doing this. Those workflows to me seem more like a barrier than any form of trying to integrate and follow what was outline from the start.

**Comment by sepu85** [comment]
It's about weighting doing the right thing over doing the things right... 

I think we can focus on the building the Purpose @calcadarafael . If the workflow is a barrier or not, I guess we will see very soon.

**Comment by AaronAbuUsama** [comment]
@calcadarafael 
> I see no reason why the AGP-40 that has "deliverables" is more important than the WHY we are doing this

1. No one said the why isn't important

2. AGP40 deliverabled are a priority because the coop has a fiduciary responsibility to fulfill them

3. Because the DAO voted this to be official policy

**Comment by sepu85** [comment]
> * AGP40 deliverabled are a priority because the coop has a fiduciary responsibility to fulfill them
> *

yes, we have the fiduciary responsability to fulfill them, but that doesn't make it a priority if we actually find the "why" an important mean (/component) to reach the end in adding quality and sense to the AGP deliverables themself

> 3\. Because the DAO voted this to be official policy

that's right, but it did it without giving time to other proposals to appear (kind of the fastest solution to a crisis, call it a patch, or a 'workarround' instead of a solid solution if you want to..)

it's ok to have people like @burrrata and you @pythonpete32 focused on the here and now, but you need also the counterposition of people focusing in the long term, @calcadarafael is doing that, as myself as well..

Let's just allow ourselves to steer the Coop into the same direction, even if in the short term some seems, and repeat, seems, to be against each other. 

Good call today, sorry if I had to leave soon.
